src/Tools/onnx-mlir-opt/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_onnx_mlir_executable(onnx-mlir-opt
   LINK_LIBS PRIVATE
   ${OMLibs}
   CompilerOptions
+  Accelerator
```

Comment:
Do we need to add this? I see that you make InitAcelerator link to Accelerator.

Reply:
I will revert the change in InitAccelerator and avoid linking the Accelerator library because it already link the NNPAAccel.